packages/shared/src/utils/normalizers/zero-dimensions.js

```diff
@@ -43,6 +44,8 @@ export default function normalizeZeroDimensions(
       node.value = '0deg';
     } else if (timings.indexOf(dimension.unit) !== -1) {
       node.value = '0s';
+    } else if (dimension.unit === fraction) {
+        node.value = '0fr';
```

Comment:
Would you mind updating this to account for `0%` as well?

Reply:
Done!